docusaurus/docs/reference/ha/bootstrapping.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: Bootstrapping
+sidebar_position: 10
+---
+
+# Bootstrapping A Cluster 
+
+To bring up a controller cluster, one starts with a single node. 
+
+## Controller Configuration
+
+### Certificates
+
+Each controller requires appropriate certificates. The certificates for clustered controllers 
```

Comment:
As this appears to be a pre-requisite, it feels like it should be part of the bootstrapping section. Maybe it makes sense to turn this into a group and have the steps of bootstrapping called out as individual left-nav sections.

Bootstrapping
* creating certs
* update config file
* run controller
* initialize controller
* add nodes
* removing nodes

Also this all seems like 'operations' to me. Looking at the 'Operations' page, other than restoring backup and open ports, it seems more like it might be a 'monitoring' type of page?

Feels like grouping some of this would help.  Maybe "Configuring" vs "Running|Monitoring"?

Reply:
Moving add/remove nodes to operations. Splitting the rest into;
*overview
* certs
* config
* initialization